authentication/serializers.py

```diff
@@ -89,7 +89,7 @@ def update(self, instance, validated_data):
             instance = super().update(instance, validated_data)
             Utils.send_verification_email(get_current_site(self.context['request']).domain, instance, access_token)
             logger.info(
-                    f"User {instance.email} {instance.first_name} {instance.surname} updated his\
+                f"User {instance.email} {instance.first_name} {instance.surname} updated his\
                      account information:{validated_data} (Email in process of verification)")
```

Comment:
As we discussed earlier, each application should be refactored separately. Each engineer is responsible for their code. So you don't need to refactor every file in the project.